dockers/Screenshotter/screenshotter.js

```diff
@@ -80,6 +83,10 @@ const opts = require("nomnom")
         flag: true,
         help: "With `--verify`, produce image diffs when match fails",
     })
+    .option("regenerate", {
+        flag: true,
+        help: "With `--verify`, generate new screenshots when match fails",
+    })
```

Comment:
I wonder if this option would be better named `keepnew`/`keepNew` or `savenew`/`saveNew` or something similar.  I would have guessed `regenerate` would overwrite the existing image, like the default behavior, but it instead saves it in a `new` directory.

Reply:
@edemaine I think these tools should not overwrite files without a prompt or user's explicit informed approval. For its name, I'm open to suggestions 😄 Maybe we can print out where diffs and/or new screenshots have been generated.

Reply:
Agreed, I'm happy it doesn't overwrite!

What do you think about a `--new` flag, given that the directory name is called `new`, just like `--diff` and `diff`?  But maybe it's also ambiguous.  My favorite other name is probably `--savenew`...

I like the idea of printing out the directories, if possible.  It would also be nice to mention the directories in the option docstrings.

Reply:
@edemaine Renamed to `new`.